src/test/java/Homework17.java

```diff
@@ -1,30 +1,47 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
-public class Homework17 extends BaseTest {
-
+public class Homework17 extends BaseTest{
     @Test
-    public void addSongToPlaylist() throws InterruptedException {
-        login("demo@class.com", "te$t$tudent");
-        Thread.sleep(2000);
-        grabASong();
+    public static void addSongtoPlaylist() throws InterruptedException {
+
+        navigateToPage();
+        Thread.sleep(1000);
+        login("ainurbilyalova818@gmail.com", "te$t$tudent");
+        Thread.sleep(1000);
+        getASong();
+
+
+        WebElement song = driver.findElement(By.xpath("//section[@id='playlistWrapper']/div[@class='song-list-wrap main-scroll-wrap playlist']/div[@class='virtual-scroller scroller']/div[@class='item-container']/table[@class='items']/tr[@class='song-item']/td[contains(text(), 'Waiting On A Train')]"));
```

Comment:
Try to write a shorter locators, if you are usig some uniqe text (example: Waiting On A Train) you can make your locator mush shorter and easy to read. 

In this case it can be done like this -  (//td[contains(text(), 'Waiting On A Train')])[1]

Reply:
Hi, KirilI!
thank you for the helpful advice!